ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/PhotoEdit.java

```diff
@@ -13,55 +13,50 @@
 public class PhotoEdit {
```

Comment:
I agree, this implementation needs refactoring. I will add it to the todos for this project.